examples/bugzilla_examples.md

```diff
@@ -88,6 +88,9 @@ t = BugzillaTicket(<bugzilla_url>,
 t.add_comment('Test Comment')
 t.edit(priority='low',
        severity='low')
+t.add_attchment(file_name='test_attachment.patch',
+                data=<contents/file-location>,
```

Comment:
I think leaving it to <contents/file-location> will be a better since the user has an option to pass 'string' as well as 'file-location'